templates/cluster-template-crs-cni.yaml

```diff
@@ -140,18 +141,16 @@ spec:
           provider-id: equinixmetal://{{ `{{ v1.instance_id }}` }}
     postKubeadmCommands:
     - |
-      cat <<EOF >> /etc/network/interfaces
-      auto lo:0
-      iface lo:0 inet static
-        address {{ .controlPlaneEndpoint }}
-        netmask 255.255.255.255
-      EOF
-    - systemctl restart networking
+      if [ -f "/run/kubeadm/kubeadm-join-config.yaml" ]; then
+        KVVERSION=$(curl -sL https://api.github.com/repos/kube-vip/kube-vip/releases | jq -r ".[0].name")
```

Comment:
Can you hard code the version of kube-vip? We want to test new kube-vip versions before we update cluster-api to start using them in the future.

Reply:
Sure. Just simply hardcore it or should I expose it through and env var and have there be a default value?

Reply:
I've just pinned the kube-vip version so I'm resolving this thread.